pkgs/applications/networking/feedreaders/rsstail/default.nix

```diff
@@ -21,15 +20,16 @@ stdenv.mkDerivation {
   makeFlags = [ "prefix=$(out)" ];
   enableParallelBuilding = true;
 
-  doCheck = true;
+  # just runs cppcheck linter
+  doCheck = false;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** 

Sorry I'm newbie here, but sometimes we set it to false (this), sometimes isn't defined (migraphx), and in sqlitecpp is true.
migraphx will check it?
sqlitecpp will not fail?